lib/transport_web/controllers/user_controller.ex

```diff
@@ -21,6 +23,41 @@ defmodule TransportWeb.UserController do
     end
   end
 
+  def organization_form(conn, _params) do
+    render conn, "organization_form.html"
+  end
+
+  def organization_create(conn, %{"name" => name,
```

Comment:
Validation logic shouldn't be controller's responsibility. That should probably go into a struct or another data structure.